Developed a voice-to-text application that supports both microphone…

Vishal-sundar-kaira opened the pull request:
… input and file uploads.In this project I started by reading documentation(charlywargnier post)provided to us and then using youtube platform made voice to text using hugging face library Web2Vec2 and used pretrained models (explained in code) and completed implementing and then launched it in streamlit by reading their documentations. Now to improve this project i used chatgpt and added features of microphone to directly speak and file(.wav) will be created automatically and stored using soundfiles and different libraries are used.It took me around 7 hours to complete this project due to hackthons and its project i was unable to give more time and add more features.I enjoyed working on my first project related to using pretrained models in python and looking forward to add more features in voicetotext project.

anubrag commented:
Hey @Vishal-sundar-kaira  , Interesting . 
Could you please attempt to debug the code and tell me the results?  Additionally, if you have any hypotheses regarding any potential issues, please share them as well.

Vishal-sundar-kaira commented:
Sorry sir at first i added different file where my explanation how i made it was not present now i updated my python file.Can you specify what issue you are facing so that i can debug it.It working correctly in my device.
